src/nonLinPMMH.cpp

```diff
@@ -97,6 +99,14 @@ Rcpp::DataFrame nonLinPMMH_impl(arma::vec data, unsigned long lNumber, unsigned
                 loglike(i) = loglike(i-1);
                 logprior(i) = logprior(i-1);
             }
+            if (i % 100 == 0 && verbouse){
+		Rcpp::Rcout << "========================="<< endl;
```

Comment:
This output looks a bit longer and more cluttered than is really needed, wouldn't:
`"Iteration: " << i << " Average acceptance probability: " << MH_ratio_total / i << ...`
suffice without taking half a dozen lines to output three or four numbers?